src/tools/perf/perftest.c

```diff
@@ -219,16 +220,19 @@ static void sock_rte_barrier(void *rte_group, void (*progress)(void *arg),
 #pragma omp master
   {
     sock_rte_group_t *group = rte_group;
-    const unsigned magic = 0xdeadbeef;
-    unsigned snc;
 
-    snc = magic;
-    safe_send(group->connfd, &snc, sizeof(unsigned), progress, arg);
+    if (group->size > 1) {
```

Comment:
An alternative: if group->size < 2, return.

Less indentation (the big block won't indent).

Reply:
but in current the logic is more clear

Reply:
+1
if (group_size <= 1) { return; }

Reply:
Unresolved

Reply:
still don't agree

Reply:
looking again at the code, maybe it's not really possible due to openmp pragmas